mslice/presenters/cut_plotter_presenter.py

```diff
@@ -86,9 +87,10 @@ def get_icut(self, workspace_name):
         except KeyError:
             return None
 
-    def update_main_window(self):
-        self._main_presenter.highlight_ws_tab(2)
-        self._main_presenter.update_displayed_workspaces()
+    def update_main_window(self, is_gui=True):
+        if is_gui:
```

Comment:
Is it possible to make the `_main_presenter` optional so we just have `None` and check if `self._main_presenter` is `None`?